README.md

```diff
@@ -36,8 +36,9 @@ to Mac users, but Lima can be used for non-container applications as well.
 
 ## Adopters
 
-- [Rancher Desktop](https://rancherdesktop.io/): Kubernetes and container management to the desktop
 - [Colima](https://github.com/abiosoft/colima): Docker (and Kubernetes) on macOS with minimal setup
+- [Lima xbar plugin](https://github.com/unixorn/lima-xbar-plugin) - Plugin for [xbar](https://xbarapp.com/) to create a Lima menu in your macOS menubar to start/stop your Lima VMs and see their running status.
```

Comment:
Seems slightly verbose compared to the other entries; it is kind of implied that this is about Lima. 😄 

```suggestion
- [Lima xbar plugin](https://github.com/unixorn/lima-xbar-plugin) - [xbar](https://xbarapp.com/) plugin to start/stop VMs from the menu bar and see their running status.
```

On second thoughts, this is just nit-picking, and the existing entry is fine too.